homepage.css

```diff
@@ -10,7 +10,8 @@
 :root {
   --gray-shade1: #f5f5f6;
   --gray-shade2: #696e79;
-  --grey-shade3: #495057;
+  --gray-shade3: #495057;
+  --gray-shade4: #f1f3f6;
```

Comment:
numbers are not understable for colors, you can use like dark, darker, shade, light, etc